fix(docs): Update troubleshooting and include docker commands (#36070)
freeCodeCamp/freeCodeCamp: docs/how-to-setup-freecodecamp-locally.md

There might be an error in the console of your browser or in Bash / Terminal / Command Line that will help identify the problem. Provide this error message in your problem description so others can more easily identify the issue and help you find a resolution.

### Troubleshooting

> If the app launches but you are encountering errors with the UI itself, for example if fonts are not being loaded or if the code editor is not displaying properly, try the following if you are using docker:

Reviewer: Just changing some of the phrasing as well as separating the different methods more clearly. This makes it slightly easier to follow along.

```suggestion
Upon launching the app, if you are encountering issues with the UI such as the fonts not loading or the code editor not displaying, try the following depending on your situation.

#### Using Docker
```

Reviewer 2: This suggestion is not proper grammar and definitely does not feel like it makes it easier to read in my opinion.  It is fine starting with "If" and not fine to start with a prepositional phrase like "Upon launching the app".  How about something more like:
> If the app launches but you are encountering UI errors such as fonts not being loaded or the code editor not displaying properly, see the following depending on your local setup:

### Using Docker:

```sh
# We use a mono repo and have multiple components (server, client, tools, plugins, etc.)
# Use this command to clean up all dependencies in all of the components
npm run docker:clean

# Reinstall npm packages
npm run docker:install

# Seed the database
npm run docker:seed

# Restart the application
npm run docker:develop
```

> If you are not using docker, the process is very similar, but uses different versions of each command:

Reviewer: ```suggestion
### Without Docker
```

Reviewer 2: Continuing the flow from above:
```suggestion
Not Using Docker
```
